lib/cf_app_discovery/client.rb

```diff
@@ -17,15 +17,14 @@ def apps
       end
 
       resources.each do |resource|
-        resource[:route] = get_first_route(resource)
+        add_first_route(resource)
       end
       resources
     end
 
     def app(app_guid)
       resource = get("/v2/apps/#{app_guid}")
-      resource[:route] = get_first_route(resource)
-      resource
+      add_first_route(resource)
```

Comment:
this isn't quite what I intended... let's chat tomorrow

Reply:
we spoke irl and it turns out that what I intended turned out not to be possible